modules/hmftools/1.0/hmftools.smk

```diff
@@ -0,0 +1,631 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
```

Comment:
please describe this file in the changelog

Reply:
Sorry but I don't see where you described the hmftools (non-GRIDSS) part of this in the Changelog